recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
-{% set version = "2.5.3" %}
+{% set version = "2.5.4" %}
 
 package:
   name: magma
   version: {{ version }} 
 
 source:
    url: http://icl.utk.edu/projectsfiles/magma/downloads/magma-{{ version }}.tar.gz
-   sha1: 809867a477d67d0b137f6b2cea4eecaec057f53c 
+   sha1: c428c9486c5f46f3595570ec898274a8bb888d5c
    patches:
      - cmakelists.patch
```

Comment:
Let's organize these patches like we have for some of the other recipes (like tensorflow) into categories.